pkg/operator/k8sutil/pod.go

```diff
@@ -118,6 +118,32 @@ func GetSpecContainerImage(spec v1.PodSpec, name string, initContainer bool) (st
 	}
 	return image.Image, nil
 }
+func AddUnreachableNodeToleration(podSpec *v1.PodSpec, tolerationSeconds int64) {
```

Comment:
I would think we can use a constant setting for tolerationSeconds. How about if we just define it inside this method as 5s and remove parameter to this method?

Reply:
+1 unless we would want to have a different timeout depending on the daemon.

Reply:
@Travisn: Good idea!!,  and even better if we use a configurable value. (to avoid the problem exposed by @BlaineEXE ).
I have added the operator var "ROOK_UNREACHABLE_NODE_TOLERATION_LAPSE" for allowing users to configure this value.
@leseb : I think it will be ok and we won't need to adjust this setting depending of the daemon. In any case, a more sophisticated implementation always is possible.